site/react/createConfig.md

```diff
@@ -58,16 +58,6 @@ import { type AlchemyAccountsConfig } from "@alchemy/aa-alchemy/config";
 
 Returns an object containing the Alchemy Accounts state.
 
-### bundlerClient
-
-`ClientWithAlchemyMethods`
-A JSON RPC client used to make requests to Alchemy's Nodes and Bundler.
-
-### signer
-
-`AlchemySigner`
-The underlying signer instance used by Embedded Accounts. This property is only available on the client.
```

Comment:
let's pretend this didn't happen >.>

Reply:
reason for that is, most of the properties on this type are incredibly unlikely to be used by anyone. We have `get*` methods for reading state from a config and the hooks all wrap those methods. For like 99.9% of people this isn't a breaking change.

if you're in .1%... I'm sorry